hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/test/java/org/apache/hadoop/yarn/client/cli/TestSchedConfCLI.java

```diff
@@ -154,7 +155,8 @@ private static void setupQueueConfiguration(
     config.setMaximumCapacity(a, 100f);
   }
 
-  private void cleanUp() throws Exception {
+  @After
```

Comment:
Since this is `@After` now, it would end up getting triggered twice for methods that directly invoke it: `testGetSchedulerConf` and `testFormatSchedulerConf`. Can you please remove those two direct calls?